python/private/repo_utils.bzl

```diff
@@ -98,6 +98,8 @@ def _execute_internal(
         arguments,
         environment = {},
         logger = None,
+        log_stdout = True,
+        log_stderr = True,
```

Comment:
Alternative API option (but naming is hard):

```python
log_std = "both"  # one of ["both", "stdout", "stderr"]
```

(I'd prefer an enum but this is starlark not python :melting_face:)

LMK which API people would prefer.

Reply:
I think the log_xxx API is fine. These are internal helpers, so easy to change if we want.